solution/ui/regulations/css/scss/_application_settings.scss

```diff
@@ -192,6 +197,15 @@ $font_size_xxl: 30px; // Aligns with the CMS Design System, but their variable i
 $font_size_hero_h1: 26px;
 $font_size_hero_h1_tablet: 20px;
 
+@mixin font-bold-keep-width {
+    text-shadow: -0.06ex 0 0 currentColor, 0.06ex 0 0 currentColor;
+
+    @supports (-webkit-text-stroke-width: 0.04ex) {
+        text-shadow: -0.03ex 0 0 currentColor, 0.03ex 0 0 currentColor;
+        -webkit-text-stroke-width: 0.1ex;
+    }
```

Comment:
grabbed [this trick from StackOverflow](https://stackoverflow.com/questions/5687035/css-bolding-some-text-without-changing-its-containers-size/46452396#46452396) to keep character width the same while still adding font weight.